src/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant.Tests/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../GSoft.Authentication.ClientCredentialsGrant.snk</AssemblyOriginatorKeyFile>
+    <RootNamespace>GSoft.Extensions.Http.Authentication.ClientCredentialsGrant.Tests</RootNamespace>
```

Comment:
`RootNamespace` has been added when renaming the project, it still refers to the previous namespace, we should delete this line

Reply:
Will do!